django_redis/cache.py

```diff
@@ -61,6 +62,7 @@ def __init__(self, server, params):
             if self._log_ignored_exceptions
             else None
         )
+        self._dj_ver = DJANGO_VERSION[:3]
```

Comment:
No need to store this value anywhere, it can be mocked in tests

Reply:
done.

```
   @patch("django_redis.cache.DJANGO_VERSION", (3, 0, 1, 'final', 0))
    def test_delete_return_value_type_before31(self):
```